### PR TITLE
doc: update Log4j2 configuration and gremlin-console guide

### PR DESCRIPTION
**Pull request by VGalaxies**
When starting HugeGraphServer with the script or with IDEA, the log4j configuration file being read is not the same file, which may cause some confusion for new developers. Therefore, I have added this part of the content to clarify it.

The gremlin-console guide is for PR https://github.com/apache/incubator-hugegraph/pull/2231

> EN doc will be synchronized later.

### CODE REVIEW

File: content/cn/docs/clients/gremlin-console.md [hunk 1]
```diff
@@ -4,305 +4,203 @@ linkTitle: "Gremlin Console"
...
+schema.propertyKey("age").asInt().ifNotExist().create()
+schema.propertyKey("city").asText().ifNotExist().create()
+schema.propertyKey("weight").asDouble().ifNotExist().create()
+schema.propertyKey("lang").asText().ifNotExist().create()
+schema.propertyKey("date").asText().ifNotExist().create()
+schema.propertyKey("price").asInt().ifNotExist().create()
+
+schema.vertexLabel("person").properties("name", "age", "city").primaryKeys("name").ifNotExist().create()
+schema.vertexLabel("software").properties("name", "lang", "price").primaryKeys("name").ifNotExist().create()
+// schema.indexLabel("personByName").onV("person").by("name").secondary().ifNotExist().create()
```
Comment (imbajin) [inline content/cn/docs/clients/gremlin-console.md:44]: useless?

File: content/cn/docs/clients/gremlin-console.md [hunk 2]
```diff
@@ -4,305 +4,203 @@ linkTitle: "Gremlin Console"
...
+
+System.out.println(">>>> query all vertices: size=" + g.V().toList().size())
+System.out.println(">>>> query all edges: size=" + g.E().toList().size())
 ```
 
-其实这一段groovy脚本几乎就是Java代码，不同之处仅在于变量的定义可以不写类型声明，以及每一行末尾的分号可以去掉。
+其实这一段 groovy 脚本几乎就是 Java 代码，不同之处仅在于变量的定义可以不写类型声明，以及每一行末尾的分号可以去掉。
 
-> g.V() 是获取所有的顶点，g.E() 是获取所有的边，toList() 是把结果存到一个 List 中，参考[TinkerPop Terminal Steps](http://tinkerpop.apache.org/docs/current/reference/#terminal-steps)。
+> `g.V()` 是获取所有的顶点，`g.E()` 是获取所有的边，`toList()` 是把结果存到一个 List 中，参考 [TinkerPop Terminal Steps](http://tinkerpop.apache.org/docs/current/reference/#terminal-steps)。
```
Comment (imbajin) [inline content/cn/docs/clients/gremlin-console.md:78]: nice improvement

File: content/cn/docs/clients/gremlin-console.md [hunk 3]
```diff
@@ -4,305 +4,203 @@ linkTitle: "Gremlin Console"
...
+==>[id:2:lop,label:software,type:vertex,properties:[name:lop,lang:java,price:328]]
+==>[id:1:josh,label:person,type:vertex,properties:[name:josh,age:32,city:Beijing]]
+==>[id:1:marko,label:person,type:vertex,properties:[name:marko,age:29,city:Beijing]]
+==>[id:1:peter,label:person,type:vertex,properties:[name:peter,age:35,city:Shanghai]]
+==>[id:1:vadas,label:person,type:vertex,properties:[name:vadas,age:27,city:Hongkong]]
+==>[id:2:ripple,label:software,type:vertex,properties:[name:ripple,lang:java,price:199]]
 ```
 
-在Client/Server模式下，所有跟Server有关的操作都要加上`:> `，如果不加，表示在console本地操作。
+在 Client/Server 模式下，所有和 Server 有关的操作都要加上 `:> `，如果不加，表示在 console 本地操作。
```
Comment (imbajin) [inline content/cn/docs/clients/gremlin-console.md:186]:
```suggestion
Note: 在 Client/Server 模式下，所有和 Server 有关的操作都要加上 `:> `，如果不加，表示在 console 本地操作。
```

File: content/cn/docs/quickstart/hugegraph-server.md [hunk 4]
```diff
@@ -431,4 +431,61 @@ $bin/stop-hugegraph.sh
...
+
+2023-06-10 19:41:14 [main] [INFO] o.a.h.u.ConfigUtil [org.apache.hugegraph.util.ConfigUtil.scanGraphsDir(ConfigUtil.java:88)] - Scanning option 'graphs' directory './conf/graphs'
+2023-06-10 19:41:14 [main] [INFO] o.a.h.d.HugeGremlinServer [org.apache.hugegraph.dist.HugeGremlinServer.start(HugeGremlinServer.java:52)] - Configuring Gremlin Server from /Users/dingyuchen/Desktop/hugegraph/apache-hugegraph-incubating-1.0.0/conf/gremlin-server.yaml
+>>>> query all vertices: size=6
+>>>> query all edges: size=6
+```
+
+并且使用 RESTful API 请求 `HugeGraphServer` 得到如下结果：
+
+```json
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-server.md:480]:
```suggestion
```javascript
```

**Review by imbajin — COMMENTED** [review]
Thanks, could also update the title of [IDEA config](https://github.com/apache/incubator-hugegraph-doc/blob/master/content/en/docs/contribution-guidelines/hugegraph-server-idea-setup.md) doc, reduce the title to `setup server in IDEA`

image

**Comment by imbajin** [comment]
And a small tips:
we could paste the url directly if it belongs to the GitHub itself, and the UI will be more readable

image

**Comment by VGalaxies** [comment]
> A small tips: we could paste the url directly if it belongs to the GitHub, and the UI will be more readable
> 
> image

Understood, I will take note of it.

File: content/cn/docs/clients/gremlin-console.md (hunk 1 above)
Comment (VGalaxies) [reply content/cn/docs/clients/gremlin-console.md]: Removed now.

**Comment by VGalaxies** [comment]
> Thanks, could also update the title of [IDEA config](https://github.com/apache/incubator-hugegraph-doc/blob/master/content/en/docs/contribution-guidelines/hugegraph-server-idea-setup.md) doc, reduce the title to `setup server in IDEA`
> 
> image

Done.

**Review by javeme — APPROVED** [review]
LGTM

**Comment by VGalaxies** [comment]
> enhance it next time
> 
> image
> 
> * add number ①②... for the problems)
> * better to add a **separate line** before the reference

Got it.

**Comment by imbajin** [comment]
Also find some outdated configs:

image

- https://hugegraph.apache.org/docs/config/config-guide/

File: content/en/docs/quickstart/hugegraph-server.md [hunk 5]
```diff
@@ -441,4 +441,51 @@ $bin/stop-hugegraph.sh
 
 ### 8 Debug Server with IntelliJ IDEA
 
-Please refer to [How to Set Up HugeGraph-Server Development Environment in IDEA](/docs/contribution-guidelines/hugegraph-server-idea-setup)
\ No newline at end of file
+Please refer to [Setup Server in IDEA](/docs/contribution-guidelines/hugegraph-server-idea-setup)
+
+### 9 Create Sample Graph on Server Startup
```
Comment (imbajin) [inline content/en/docs/quickstart/hugegraph-server.md:446]: we need intro the basic arch for server first to let users know how to config it clearly, like the question:

image


maybe enhance it in next PR together

**Comment by msgui** [comment]
> Also find some outdated configs:
> 
> image
> * [hugegraph.apache.org/docs/config/config-guide](https://hugegraph.apache.org/docs/config/config-guide/)
> 
> image - https://hugegraph.apache.org/cn/docs/introduction/readme/

Hey @imbajin, I've just updated the outdated configs in pr #272.

**Pull request merged**